File: 2586-count-the-number-of-vowel-strings-in-range/2586-count-the-number-of-vowel-strings-in-range.py
class Solution:
    def vowelStrings(self, words: List[str], left: int, right: int) -> int:
        count = 0
        vowels = {'a', 'e', 'i', 'o', 'u'}
        for ele in range(left,right+1):
            if words[ele][0] in vowels and words[ele][-1] in vowels:
                count +=1
        return count

File: 2586-count-the-number-of-vowel-strings-in-range/2586-count-the-number-of-vowel-strings-in-range.java
class Solution {
    public int vowelStrings(String[] words, int left, int right) {
        char one , two;
		int output = 0;
		for (int i=left;i<=right;i++) {
		    one = words[i].charAt(0);
		    two = words[i].charAt(words[i].length()-1);
		    if((one=='a'||one=='e'||one=='i'||one=='o'||one=='u')&&(two=='a'||two=='e'||two=='i'||two=='o'||two=='u'))
		        output++;
		}     
        return output;     
    }
}